GameServer/Logic/GameObjects/AzirTurret.cs

```diff
@@ -0,0 +1,209 @@
+﻿using InibinSharp;
+using LeagueSandbox.GameServer.Core.Logic;
+using LeagueSandbox.GameServer.Core.Logic.RAF;
+using LeagueSandbox.GameServer.Logic.Enet;
+using LeagueSandbox.GameServer.Logic.Maps;
+using LeagueSandbox.GameServer.Logic.Packets;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LeagueSandbox.GameServer.Logic.GameObjects
+{
+    public class AzirTurret : Unit
+    {
+        private const float TURRET_RANGE = 905.0f;
+        private string _name;
+        private Unit _owner;
+        private float _globalGold = 250.0f;
+        private float _globalExp = 0.0f;
+
+        public AzirTurret(Game game, Unit owner, uint id, string name, string model, float x = 0, float y = 0, TeamId team = TeamId.TEAM_BLUE) : base(game, id, model, new Stats(), 50, x, y, 1200)
```

Comment:
120 character limit, although might be hard to comply here. Would move the ` : base(...);` part to a new line, though.